site/snippets/sidebar.php

```diff
@@ -0,0 +1,38 @@
+
+<aside>
+  <p class="intro"><?php echo $site->description()->kirbytext() ?></p>
+  <a class="skip-link visuallyhidden focusable" href="#main">Skip to Main Content</a>
+  <nav class="sidebar-nav" role="navigation">
+    <ul>
+      <?php foreach($pages->visible() as $p): ?>
```

Comment:
I tried setting this to:

```php
$pages->findByURI('/containerpilot/docs')->visible()
```

That made the sidebar work, but the content failed to display.

That hack wouldn't be very portable anyway, so it's possible this site needs a a different theme/sidebar.